packages/zapp/console/src/components/Entities/EntityVersionDetails.tsx

```diff
@@ -0,0 +1,96 @@
+import * as React from 'react';
+import { withRouteParams } from 'components/common/withRouteParams';
+import { ResourceIdentifier, ResourceType } from 'models/Common/types';
+import { makeStyles, Theme } from '@material-ui/core/styles';
+import { WaitForData } from 'components/common/WaitForData';
+import { useProject } from 'components/hooks/useProjects';
+import { StaticGraphContainer } from 'components/Workflow/StaticGraphContainer';
+import { WorkflowId } from 'models/Workflow/types';
+import { entitySections } from 'components/Entities/constants';
+import { EntityDetailsHeader } from 'components/Entities/EntityDetailsHeader';
+import { EntityVersions } from 'components/Entities/EntityVersions';
+import { typeNameToEntityResource } from './constants';
+
+const useStyles = makeStyles((theme: Theme) => ({
+  verionDetailsContatiner: {
+    display: 'flex',
+    flexDirection: 'column',
+    flexWrap: 'nowrap',
+    overflow: 'hidden',
+    height: `calc(100vh - ${theme.spacing(1)}rem)`,
```

Comment:
nit: not sure we should use rem here, as most of our values in px and specing(1) assumes px